### ISSUE TITLE
can not open '_mask.pyx'

### ISSUE THREAD

**Issue by jinyuan30** [pytorch/vision#1292]
I'm doing Mask RCNN detection task, and run the case of trochvison.
I've installed pycocotool, I don't konw the detail, so could any one help me solve this bug?

**Comment by jinyuan30** [comment]
'''
Epoch: [0] Total time: 0:00:27 (0.4521 s / it)
creating index...
index created!
'''

when the Epoch[0] training done, there is the error: can not open '_mask.pyx'

**Comment by fmassa** [comment]
@jinyuan30 there is no `_mask.py`in torchvision, it is in [`pycocotools`](https://github.com/cocodataset/cocoapi/tree/master/PythonAPI/pycocotools), so I think your installation of pycocotools is not working properly.

Given that this is not a problem with torchvision per se, I'm closing this issue. I'd recommend trying to uninstall and reinstall pycocotools, and see if you can import it.

Installing pycocotools can be done as follows:
```bash
# install pycocotools
cd $INSTALL_DIR
git clone https://github.com/cocodataset/cocoapi.git
cd cocoapi/PythonAPI
python setup.py build_ext install
```
